schedule/qam/common/qam-yast_self_update_libyui-rest.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: qam-yast_self_update_libyui-rest
+description: installation using self_update as boot parameter
+vars:
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/validate_self_update
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_modules
+  - installation/add_on_product/skip_install_addons
```

Comment:
we don't skip addons in maintenance, we need still needle-based module `installation/add_update_test_repo`